proposals/primer/snippets/prefixes.ttl

```diff
@@ -4,9 +4,23 @@
 @prefix dcterms: <http://purl.org/dc/terms/> .
 @prefix solid: <http://www.w3.org/ns/solid/terms#> .
 @prefix interop: <http://www.w3.org/ns/solid/interop#> .
+
```

Comment:
Should also include the prefix for shape trees. We typically are using:

`@prefix st: <http://www.w3.org/ns/st#>`

Reply:
added `@prefix shapetree: <http://www.w3.org/ns/shapetree#> .` as used in all the snippets.